mappings/net/minecraft/block/FlowerPotBlock.mapping

```diff
@@ -6,4 +6,5 @@ CLASS net/minecraft/unmapped/C_urzerdak net/minecraft/block/FlowerPotBlock
 	METHOD <init> (Lnet/minecraft/unmapped/C_mmxmpdoq;Lnet/minecraft/unmapped/C_triydqro$C_xnkxsdfy;)V
 		ARG 1 content
 		ARG 2 settings
+	METHOD m_ckgcbvvg isPotted ()Lnet/minecraft/unmapped/C_mmxmpdoq;
```

Comment:
```suggestion
	METHOD m_ckgcbvvg getContents ()Lnet/minecraft/unmapped/C_mmxmpdoq;
```
the `is` prefix is incorrect here, that should only be for booleans. I'd rename the automapped field to `contents` as well

Reply:
```suggestion
	METHOD m_ckgcbvvg getPotted ()Lnet/minecraft/unmapped/C_mmxmpdoq;
```
to match the string?